# This patch is removing a gresource since the svg file is stripped for licensing reason.
# This resource is not used elsewhere in the application.
--- a/data/com.github.bilelmoussaoui.Authenticator.gresource.xml
+++ b/data/com.github.bilelmoussaoui.Authenticator.gresource.xml
@@ -5,7 +5,6 @@
     <file>data.json</file>
     <file alias="qrscanner-symbolic.svg">icons/hicolor/symbolic/actions/qrscanner-symbolic.svg</file>
     <!-- Accounts logo fallback -->
-    <file alias="authenticator-symbolic.svg">icons/hicolor/symbolic/apps/com.github.bilelmoussaoui.Authenticator-symbolic.svg</file>
     <file alias="authenticator.svg">icons/hicolor/scalable/apps/com.github.bilelmoussaoui.Authenticator.svg</file>
 
     <!-- UI Files -->
--- a/data/icons/meson.build
+++ b/data/icons/meson.build
@@ -7,11 +7,5 @@
               install_dir: dest_scalable,
               rename: '@0@.svg'.format(application_id))
 
-  symbolic_dir = join_paths(theme, 'symbolic/apps/')
-  symbolic_icon = join_paths(symbolic_dir, meson.project_name() + '-symbolic.svg')
-  dest_symbolic = join_paths(get_option('prefix'), 'share/icons', symbolic_dir)
-  install_data(symbolic_icon,
-              install_dir: dest_symbolic,
-              rename: '@0@'.format(application_id) + '-symbolic.svg')
 endforeach
 
